rpc/gnmi/gnmi-depth.md

```diff
@@ -0,0 +1,271 @@
+# gNMI Depth Extension
```

Comment:
Can you wrap the doc to 80 chars please?

Reply:
done

Reply:
Thanks @robshakir 

I also addressed other review comments in [f28e8fa](https://github.com/openconfig/reference/pull/205/commits/f28e8faffc585824cdb2bdf968267a5c95c01f67)